### ISSUE TITLE
Ping is really low

### ISSUE THREAD

**Issue by Little-code49** [chubin/wttr.in#843]
![Screen Shot 2023-02-18 at 15 37 03](https://user-images.githubusercontent.com/76576636/219850698-6c02dc8b-83e0-4589-bd2d-d4f112d98834.png)

**Comment by obehrens** [comment]
The site is completely down for me. It responds to pings, but there is a "secure connection failed" error when trying to access it from a web browser, and when using cURL -v wttr.in, I get 
`*   Trying 5.9.243.187:80...`
`* Connected to wttr.in (5.9.243.187) port 80 (#0)`
`> GET / HTTP/1.1`
`> Host: wttr.in`
`> User-Agent: curl/7.87.0`
`> Accept: */*`
`> `
`* Empty reply from server`
`* Closing connection 0`
`curl: (52) Empty reply from server`

![Screenshot_20230218_070827](https://user-images.githubusercontent.com/118389417/219865080-94887a85-4378-4ba1-8c7d-5f38c45be0be.png)

**Comment by rahulTPatil** [comment]
Yes. Seeing the downtime from today morning. Hopefully it gets resolved soon.

**Comment by obehrens** [comment]
It's back up for me at 7:15 EST (Feb. 18)
